apps/hash-realtime/src/index.ts

```diff
@@ -66,8 +66,8 @@ const acquireReplicationSlot = async (
             -- Second argument to this calls specifies whether or not the replication
             -- slot is temporary.
             select 1 from pg_create_logical_replication_slot(${slotName}, 'wal2json', ${
-              options.temporary ?? true
-            })
+    options.temporary ?? true
+  })
```

Comment:
Do you have a different `prettier` version installed or why was this changed?

Reply:
I was wondering about that myself, but I have yarn installed repeatedly (with `just yarn install`) and I always get the same result, 4 files in actuality changed. This one slipped through, the other three I correctly stashed.

Reply:
Yea, if we look into the CI we can see:

```
Checking formatting...
[warn] apps/hash-realtime/src/index.ts
[warn] apps/hash/README.md
[warn] tests/hash-playwright/tests/shared/login-using-ui.ts
[warn] Code style issues found in 3 files. Run Prettier to fix.
```

Those are the other three files. Weird stuff.

Reply:
Well, it's good to have it fixed then I guess 😄